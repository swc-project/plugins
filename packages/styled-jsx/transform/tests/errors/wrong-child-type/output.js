import _JSXStyle from "styled-jsx/style";
export default (()=><div >

    <style jsx>

      10

    </style>

  </div>);
