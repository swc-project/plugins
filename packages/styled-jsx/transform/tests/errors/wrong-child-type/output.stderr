
  x Expected a single child of type JSXExpressionContainer under JSX Style tag.
  | Read more: https://nextjs.org/docs/messages/invalid-styled-jsx-children
   ,-[input.js:3:5]
 3 | ,-> <style jsx>
 4 | |         10
 5 | `->     </style>
   `----
